core/src/main/scala/kafka/server/ReplicationQuotaManager.scala

```diff
@@ -39,18 +39,9 @@ import org.apache.kafka.common.utils.Time
   * @param quotaWindowSizeSeconds     The time span of each sample
   *
   */
-case class ReplicationQuotaManagerConfig(quotaBytesPerSecondDefault: Long = QuotaBytesPerSecondDefault,
```

Comment:
We have new `ReplicationQuotaManagerConfig.java`.
In the same time scala version of class is here, also.

It seems like we need to keep only one of them.
I think changes related to `ReplicationQuotaManagerConfig` can be moved to another PR to simplify review and reduce changes.

Reply:
I can move them to another pr before to be merged before this one. If this will make it easier

Reply:
> If this will make it easier

It will make changes easier in my opinion.
Moving things atomically with clear changes is always better choice.

Reply:
On a second thought let's break them into another pr because I'll need them for KafkaConfig anyway later

Reply:
Don't hesitate to ping me for review.

Reply:
done here #15160